core/metacling/src/TClingMethodInfo.cxx

```diff
@@ -436,6 +502,17 @@ int TClingMethodInfo::InternalNext()
          }
       }
 
+      clang::UsingDecl* udecl =
+          llvm::dyn_cast<clang::UsingDecl>(*fIter);
+
+      if ( udecl ) {
+          // A UsingDecl potentially brings in a bunch of functions, so
+          // start an inner loop to catch them all
+          delete fUsingIter;
```

Comment:
Possible optimization: re-use existing `UsingIterator` instead of `delete`ing and `new`ing it.